### ISSUE TITLE
Improving the "number of issues" info at the top

### ISSUE THREAD

**Issue by mgifford** [mgifford/open-scans#141]
Can we get more context?

If you look here:
 https://mgifford.github.io/open-scans/reports/issues/issue-137/2026-03-13T14-35-08-792Z/report.html

Maybe we could take this block:

Total Issues
9714
Across 74 unique rules

And provide a concise list with the following:

- number of A, AA, AAA
- Number just WCAG 2.0, WCAG 2.1, WCAG 2.2 (for A & AA)
- number of accessibility best practice issues (that aren't WCAG SC violations)
- axe (without false positives)
- Other unique errors (beyond axe's errors that we know do not have false positives)
- Total Issues

It might look like:

Issues
- WCAG 2.2 A (3), AA (342), AAA (3423)
- WCAG 2.0 AA (3002), WCAG 2.1 (102), WCAG 2.2 (323)
- axe-strict (83), Best practice (2302), Other unique errors (2321)
- Total Issues (7232) Across 74 unique rules and 2 accessibility engines

This would give folks a better sense
